tests/cases/file_test.py

```diff
@@ -781,6 +784,18 @@ def testFilesystemAssetstore(self):
         copyTestFile = self._testUploadFile('helloWorld1.txt')
         self._testCopyFile(copyTestFile)
 
+        # Test unicode filenames for content disposition.  The test name has
+        # quotes, a Linear-B codepoint, Cyrllic, Arabic, Chinese, and an emoji.
+        filename = u'Unicode "sample" \U00010088 ' + \
+                   u'\u043e\u0431\u0440\u0430\u0437\u0435\u0446 ' + \
+                   u'\u0639\u064a\u0646\u0629 \u6a23\u54c1 \U0001f603'
+        file = self._testUploadFile(filename)
+        file = self.model('file').load(file['_id'], force=True)
+        testval = 'filename="%s"; filename*=UTF-8\'\'%s' % (
```

Comment:
Since this probably most useful as a regression test, maybe it's actually better to hardcode the result that we're expecting to be in `filename*`, so we implicitly test the full stack that `setContentDisposition` relies on. If you still want the more pure unit test, then you can still continue to dynamically compute the result, but compare it to the hardcoded string.

@manthey What do you think?

Reply:
I'll change this to a hardcoded value (not today, though).

Reply:
My own trial (with just `print(repr(...))`) indicates that the values may be respectively:
`'Unicode \\"sample\\"     '`
`'Unicode%20%22sample%22%20%F0%90%82%88%20%D0%BE%D0%B1%D1%80%D0%B0%D0%B7%D0%B5%D1%86%20%D8%B9%D9%8A%D9%86%D8%A9%20%E6%A8%A3%E5%93%81%20%F0%9F%98%83'`
but I'll trust however you want to encode them in this file.